jwst/lib/set_telescope_pointing.py

```diff
@@ -441,7 +453,7 @@ def update_wcs_from_telem(
          model.meta.wcsinfo.pc1_2,
          model.meta.wcsinfo.pc2_1,
          model.meta.wcsinfo.pc2_2) = calc_rotation_matrix(
-             wcsinfo.pa * D2R, vparity=siaf.vparity
+             wcsinfo.pa * D2R, model.meta.wcsinfo.v3yangle, vparity=siaf.vparity
```

Comment:
Again, convert `v3yangle` to radians before sending it along to the function.  I think this probably explains the big differences seen below in some of the tests computing the PC matrix.

Reply:
I've been forgetting that on all of the new angle args.. 🤦